### PR TITLE
Demisto class

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-2617)

## Description
Moving demisto class from server to content.

## Minimum version of Cortex XSOAR
- [x] 6.9.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
@dantavori Will update release notes only after review....

File: Documentation/common_server_docs.py [hunk 1]
```diff
@@ -147,6 +147,10 @@ def create_py_documentation(path, origin, language):
     with open(path, 'r') as file:
         py_script = YmlUnifier.clean_python_code(file.read(), remove_print_future=False)
 
+    logging.info("replacing DemistoClassApiModule: ")
+    # py_script.replace('from DemistoClassApiModule import *', '')
```
Comment (dorschw) [inline Documentation/common_server_docs.py:151]: inetnded?

File: pyproject.toml [hunk 2]
```diff
@@ -10,7 +10,7 @@ python = "^3.8,<3.11"
 blessings = "^1.7"
 coloredlogs = "^15.0.1"
 demisto_py = "^3.1.6"
-demisto_sdk= "^1.6.8"
+demisto_sdk= {git= "https://github.com/demisto/demisto-sdk.git", rev = "299b0fcc776c5623cc2b3766c6857855a98cc05d"}
```
Comment (dorschw) [inline pyproject.toml:13]: Is this part of the PR? If not, please merge from master.

Comment (daryakoval) [reply pyproject.toml]: Will be done later, this is required to run build.

File: Documentation/common_server_docs.py [hunk 3]
```diff
@@ -147,6 +147,9 @@ def create_py_documentation(path, origin, language):
     with open(path, 'r') as file:
         py_script = YmlUnifier.clean_python_code(file.read(), remove_print_future=False)
 
+    logging.info("replacing DemistoClassApiModule: ")
+    py_script = re.sub(r'from DemistoClassApiModule import \*[ \t]*(#.*)?', "", py_script)
```
Comment (dantavori) [inline Documentation/common_server_docs.py:151]: why is this required?

**Review by dantavori — APPROVED** [review]

File: Documentation/common_server_docs.py (hunk 3 above)
Comment (daryakoval) [reply Documentation/common_server_docs.py:151]: We cant import it if we want to run CommonServerPython module, it fails...

**Review by anara123 — APPROVED** [review]

**Comment by daryakoval** [comment]
Waiting to merge for demisto-sdk 1.6.9 release.
